listenbrainz/webserver/views/user.py

```diff
@@ -349,6 +350,35 @@ def collaborations(user_name: str):
     )
 
 
+@user_bp.route("/<user_name>/pins/")
+def pins(user_name: str):
```

Comment:
This endpoint should probably be paged, like accepting a count and offset in the query parameters and passing it to `get_pin_history_for_user`.